trivy-task/index.ts

```diff
@@ -196,9 +196,37 @@ function stripV(version: string): string {
 }
 
 async function getArtifactURL(version: string): Promise<string> {
-    if(version === "latest") {
+    if (version === "latest") {
+      let latestTrivyVersion = "";
+
+      try {
+        let latestTrivyVersion = await fetch(
+          new Request("https://api.github.com/repos/aquasecurity/trivy/releases/latest")
```

Comment:
This endpoint is under GitHub rate limit policy.
 It's better to parse version from `location` header of `https://github.com/aquasecurity/trivy/releases/latest`

Reply:
Good spot, thanks @bonddim